BonusCalcListener/UseCase/UpdateExistingWorkOrderPayElementsUseCase.cs

```diff
@@ -57,6 +57,14 @@ public async Task ProcessMessageAsync(EntityEventSns message)
                 return;
             }
 
+            // Ignore agency workers (those with a payroll number starting with 'T')
+            if (data.OperativePrn.StartsWith("T"))
```

Comment:
Just to confirm, this will apply to 100% of scenarios where this condition is true. Could there ever be a scenario where data.OperativePrn starts with T, and the operative is _not_ an agency worker?

Reply:
Agency workers starting with T is agreed convention